### PR TITLE
feat: add wishlist page

### PR DESCRIPTION
**Pull request by shaik-saad**
This PR includes:
- Add product images
- add footer css
- add wishlist page

Screenshots:
Screenshot 2022-02-21 014929
Screenshot 2022-02-21 015000

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.wishlist-container {
+    width: 90%;
+    margin: auto;
+}
+
+.wishlist-items-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    justify-items: center;
+    /* align-items: flex-start; */
```
Comment (Aazam10) [inline wishlist/wishlist.css]: You can remove comments if you don't need them

**Review by Aazam10 — APPROVED** [review]
The code is very clean and good use of semantic tags I have just pointed acomment which you can remove and the PR is good to merge

**Review by starver20 — APPROVED** [review]
Looks all good.

**Review by Ayushi20-19 — APPROVED** [review]
good naming conventions, and semantic tag

File: wishlist/wishlist.css (hunk 1 above)
Comment (shaik-saad) [reply wishlist/wishlist.css]: Thanks Aazam for pointing it out, removed it.

**Pull request merged**